frappe/public/js/frappe/form/toolbar.js

```diff
@@ -403,9 +403,24 @@ frappe.ui.form.Toolbar = Class.extend({
 				me.frm.page.set_view('main');
 			}, 'octicon octicon-pencil');
 		} else if(status === "Cancel") {
-			this.page.set_secondary_action(__(status), function() {
-				me.frm.savecancel(this);
-			}, "octicon octicon-circle-slash");
+			let add_cancel_button =
+				this.page.set_secondary_action(__(status), function() {
+					me.frm.savecancel(this);
+				}, "octicon octicon-circle-slash");
```

Comment:
```suggestion
			let add_cancel_button = () => {
				this.page.set_secondary_action(__(status), function() {
					me.frm.savecancel(this);
				}, "octicon octicon-circle-slash");
			}
```

Make a function... code used to execute before the check.

<img width="1440" alt="Screenshot 2020-03-31 at 6 41 15 PM" src="https://user-images.githubusercontent.com/13928957/78030888-6b370f00-7380-11ea-9315-79697ec9189e.png">